### DISCUSSION TITLE
Missing Documentation for Textarea

### DISCUSSION THREAD

**Discussion by Basti-Fantasti** [picocss/pico#630] [Help] [answered]
Hi,

I'm currently building a small HTMX website and use PICOSS.
I added a `textarea` element and can get the data written into the textare element as a value in my backend to store in the database.
But how can I set the value which read from database and handed over to the html file on the get request to the textarea field?

the `value` property as it is used on the `input` field is not working and in the documentation about the textarea this information is missing.

Any help is very appreciated

**Comment by dev-willis** [comment, marked as answer]
For `<textarea>` elements, the content goes between the opening and closing tags. There is no `value` property for this element in HTML.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/textarea

**Comment by Basti-Fantasti** [comment]
Thanks for your reply. I'll try it out right away

**Reply by Basti-Fantasti** [reply]
Just tested it and it works!
Thanks again.
